h2o-r/h2o-package/R/automl.R

```diff
@@ -138,6 +139,7 @@ h2o.automl <- function(x, y, training_frame,
                        sort_metric = c("AUTO", "deviance", "logloss", "MSE", "RMSE", "MAE", "RMSLE", "AUC", "AUCPR", "mean_per_class_error"),
                        export_checkpoints_dir = NULL,
                        verbosity = "warn",
+                       auc_type="NONE",
```

Comment:
should be:
```suggestion
                       auc_type=c("NONE", "MACRO_OVO", "WEIGHTED_OVO", "MACRO_OVR", "WEIGHTED_OVR",  "AUTO"),
```
And please see `match.arg` (https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/match.arg) or you can use case-insensitive version from explain module https://github.com/h2oai/h2o-3/blob/18341e3d3f523a01876b2c2ffa6d14fc56c79029/h2o-r/h2o-package/R/explain.R#L2-L27

Reply:
Thank @tomasfryda for the suggestion. I am still working on this PR. I converted it to a draft. I would like you to review it after I finish it. Thanks!

Reply:
I'm always happy to help with automl or R so feel free to ask if there is something unclear.